### DISCUSSION TITLE
Couldn't switch network to BNB testnet (chainId: 97)

### DISCUSSION THREAD

**Discussion by chrisll-nodereal** [trustwallet/wallet-core#3194] [Q&A]
Hi, I tried to connect Chrome extension Trust Wallet in a dApp with network switch features. I can't switch network with  wallet_switchEthereumChain method. It shows an error message  

> '1684306502099 sendError Error: Unrecognized chain ID "0x61". Try adding the chain using wallet_addEthereumChain first'.

I also tried add network with wallet_addEthereumChain, but it keeps showing a popup window below. It said the URL is invalid and doesn't allow me to add the network
![image](https://github.com/trustwallet/wallet-core/assets/104487177/461f9ca1-917e-48af-976b-d7669329d96c)

Please help me.

**To Reproduce**
```
if (typeof window !== 'undefined') {
      const { ethereum } = window;
      try {
        await ethereum?.request({
          method: 'wallet_switchEthereumChain',
          params: [
            {
              chainId: '0x61',
            },
          ],
        });
      } catch (e: any) {
        if (e?.code === 4902) {
          try {
            if (ethereum) {
              await ethereum.request({
                method: 'wallet_addEthereumChain',
                params: [
                  {
                    chainId: '0x61',
                    chainName: 'BNB Smart Chain Testnet',
                    rpcUrls: ['https://data-seed-prebsc-1-s1.binance.org:8545/'],
                    nativeCurrency: {
                      name: 'BNB',
                      symbol: 'BNB',
                      decimals: 18,
                    },
                  },
                ],
              });
            }
          } catch (addError) {
            // handle "add" error
            console.error(addError);
          }
        }
      }
    }
```

**Expected behavior**
Switch to BNB testnet in Trust Wallet

**Comment by mubshrx** [comment]
I don't think Trust Wallet supports any if the testnets.
